### ISSUE TITLE
Unknown error ?

### ISSUE THREAD

**Issue by RRadziejewski** [rancher/fleet#127]
Hello after adding repo via rancher cluster dashboard it becomes active for a moment and later goes to GIT UPDATE state with this message:

This resource is currently in an error state, but there isn't a detailed message available.

Any idea how to debug it ? this does not help much

**Comment by brodul** [comment]
I have the same issue. Is there a way to access logs?

**Comment by sks2141** [comment]
I had the same issue. I noticed that when I 'view in API' the 'GitJob' under 'Related Resources', in status field (at the very bottom of the page), it linked to the git commit id and said that the 'Stalled' state is 'true'.

I followed https://github.com/rancher/fleet/issues/93 to troubleshoot the issue. As user pcallewaert found out, I exec'ed into the gitJob container at /etc/ssh and did not notice anything to connect to private repo with the public key. I created a local ConfigMap based on directions noted by user otsuarez and patched my gitRepo deployment volume mounting the ConfigMap (data) onto /etc/ssh/ssh_known_hosts and it worked. I waited for more than 5 minutes and noticed that the 'Stalled' error has been resolved.

Hope this helps.
